py3status/modules/stopwatch.py

```diff
@@ -0,0 +1,140 @@
+# -*- coding: utf-8 -*-
+"""
+A simple stopwatch.
+
+This is a very basic stopwatch. You can start, pause and reset the
+stopwatch.
+
+Button 1 starts/pauses the stopwatch.
+Button 2 resets stopwatch.
+
+Configuration parameters:
+    format: display format for this module (default 'Stopwatch {stopwatch}')
+
+Format placeholders:
+    {stopwatch} display hours:minutes:seconds
+
+@author Jonas Heinrich
+
+SAMPLE OUTPUT
+{'full_text': 'Stopwatch 0:01:00'}
```

Comment:
No color `0:00:00`. And change `paused` `54` to `58`. Different number for screenshot.

Reply:
done :D